include/onnxruntime/core/framework/provider_options_internal.h

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#pragma once
+#define ORT_API_MANUAL_INIT
+#include "core/session/onnxruntime_c_api.h"
+
+namespace onnxruntime {
+
+// data types for execution provider options which only be used internally
```

Comment:
the name of the file doesn't really match the contents?
these aren't really provider options right?

Reply:
you are right, let me think of a proper name for this header file once we decide to add the internal header file.